src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -105,14 +105,17 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.last_changed}
                   ></ha-relative-time>
                 `
-              : this.config.secondary_info === "last-triggered" &&
-                stateObj.attributes.last_triggered
-              ? html`
-                  <ha-relative-time
-                    .hass=${this.hass}
-                    .datetime=${stateObj.attributes.last_triggered}
-                  ></ha-relative-time>
-                `
+              : this.config.secondary_info === "last-triggered"
+              ? stateObj.attributes.last_triggered
+                ? html`
+                    <ha-relative-time
+                      .hass=${this.hass}
+                      .datetime=${stateObj.attributes.last_triggered}
+                    ></ha-relative-time>
+                  `
+                : this.hass.localize(
+                    "ui.panel.lovelace.entities.never_triggered"
```

Comment:
The path you pass is still not the same as what you out in en.json

Reply:
Ive just corrected that, deleted the incorrect additions and added never_triggered to line 1700 in en.json, following Brams instruction